src/main/java/com/marklogic/kafka/connect/source/MarkLogicSourceConnector.java

```diff
@@ -71,10 +71,13 @@ public Class<? extends Task> taskClass() {
 
     @Override
     public List<Map<String, String>> taskConfigs(final int taskCount) {
-        final List<Map<String, String>> configs = new ArrayList<>(taskCount);
-        for (int i = 0; i < taskCount; ++i) {
-            configs.add(config);
+        if (taskCount > 1) {
```

Comment:
@BillFarber  @bgeorge111  This is what the user sees now if they set tasks.max to 2 or higher:

[2023-02-06 15:06:38,023] INFO [marklogic-employees-source|task-0] Starting RowManagerSourceTask (com.marklogic.kafka.connect.source.RowManagerSourceTask:67)
[2023-02-06 15:06:38,024] WARN [marklogic-employees-source|worker] As of the 1.8.0 release, the Kafka tasks.max property is ignored and a single source connector task is created. This prevents duplicate records from being created via multiple instances of the task with the exact same config. (com.marklogic.kafka.connect.source.MarkLogicSourceConnector:75)

Reply:
And the README has been updated to mention this as well.